### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by cristianetanaka**
Hi Sandra,

Footer works on Desktop and Mobile. See Link[https://shecodesaus.slack.com/archives/C06NSQTV5JR/p1712135536410319] 

Let me know if any issues. Thanks

### CODE REVIEW

File: src/components/Footer.jsx
Comment (RobyOneJ) [inline src/components/Footer.jsx:1]: @cristianetanaka - Please fix it as per Sandra's comments she sent to you in slack.

Comment (cristianetanaka) [reply src/components/Footer.jsx:1]: done - these are the results [https://shecodesaus.slack.com/archives/C06NSQTV5JR/p1712331873896529]

**Review by cristianetanaka — COMMENTED** [review]
New changes in the footer. Thank you

**Comment by Sandz-Studio** [comment]
Hi Christiane, I have redone the footer. Your tailwind was still behaving oddly when on large screens. Have a look a the new tailwind to see how I have done it.

**Review by Sandz-Studio — APPROVED** [review]

**Pull request merged**